sdk/nestjs/src/DevCycleModule/DevCycleModule.ts

```diff
@@ -0,0 +1,41 @@
+import { Module, DynamicModule, Global } from '@nestjs/common'
+import { APP_INTERCEPTOR } from '@nestjs/core'
+import { initializeDevCycle, DevCycleClient, DevCycleOptions } from '@devcycle/nodejs-server-sdk'
+import { RequestInterceptor } from './RequestInterceptor'
+
+@Global()
+@Module({
+    exports: [DevCycleClient],
+})
+export class DevCycleModule {
+    static forRoot(sdkKey: string, options?: DevCycleOptions): DynamicModule {
```

Comment:
you might wanna add a forRootAsync so people can inject the SDK key from somewhere else